pkg/controller/beat/common/stackmon/filebeat.yml

```diff
@@ -0,0 +1,8 @@
+filebeat.inputs:
+- type: filestream
+  id: filebeatlogs
+  paths:
+    - /var/log/*.log
+    - /usr/share/filebeat/data/registry/filebeat/*.json
```

Comment:
Same as for metric beat: `xpack.enabled: true` https://www.elastic.co/guide/en/beats/filebeat/current/monitoring-metricbeat-collection.html

Reply:
For metricbeat, this setting moves the metrics from being written in the `*metricbeat` indices, to the `*monitoring` indices, and allows them to show up in stack monitoring page in kibana.  This isn't true for filebeat.  I reached out to the observability team and ran it by them.  They verified no setting will allow filebeat to write data to the `*monitoring` indices, and show up in stack monitoring.

Reply:
Hm odd. I wonder why the documentation recommends setting this then.

Reply:
I just realised the documentation I mentioned covers Metricbeat monitoring Filebeat. Sorry my mistake 🤦